base/sparse/sparsematrix.jl

```diff
@@ -2942,11 +2934,11 @@ function blkdiag(X::SparseMatrixCSC...)
 end
 
 ## Structure query functions
-issymmetric(A::SparseMatrixCSC) = is_hermsym(A, IdFun())
+issymmetric(A::SparseMatrixCSC) = is_hermsym(A, identity)
 
-ishermitian(A::SparseMatrixCSC) = is_hermsym(A, ConjFun())
+ishermitian(A::SparseMatrixCSC) = is_hermsym(A, conj)
 
-function is_hermsym(A::SparseMatrixCSC, check::Func)
+function is_hermsym(A::SparseMatrixCSC, check::Function)
```

Comment:
it would be better not to use `::Function` in a signature, now that everything it callable, that distinction can be a bit fuzzy